app/views/shared/_loss_report_modal.html.erb

```diff
@@ -0,0 +1,19 @@
+<div id="loss-report-modal" class="eui-modal-content" >
```

Comment:
I was having trouble with scss to get the margins looking right on the buttons so I put the `margin-right` value directly on the rails tag. See the comment on the scss file below..

Reply:
For both of these issues, attach the css to a class and the class to the html.